src/components/common/Input/Input.tsx

```diff
@@ -21,14 +21,13 @@ const Input = ({
    */
   const toggleHiddenPassword = (event: React.MouseEvent) => {
     event.preventDefault();
-    setInputType((prevType) => (prevType === 'password' ? 'text' : 'password'));
+    setInputType((prevType) => (prevType === 'password' ? '' : 'password'));
   };
   return (
     <div className={`form-input${errors[name] ? ' error' : ''}`}>
       <label htmlFor={name}>{label} :</label>
       <div className="input-field">
         <input
-          id={name}
```

Comment:
Why is the id being removed? the label needs this on the `input` in order to let screen readers know which `input` the corresponding `label` is for